### ISSUE TITLE
Is there any way to send colored output to stderr?

### ISSUE THREAD

**Issue by renatoathaydes** [p0nce/console-colors#10]
Is there any way to send colored output to stderr?

Could that be added easily?

**Comment by p0nce** [comment]
~~Call `disableConsoleColors()` at the start of your application.~~

(aaah: you want to send coloured output to a stderr console?)

**Comment by renatoathaydes** [comment]
Yes, send output to `stderr`. There's no reason it should always send output to `stdout`?!

**Comment by renatoathaydes** [comment]
D allows you to do `stderr.writeln("foo");` so I suppose with this lib, `stderr.cwriteln("bar")` should work too.

Also, why are color names not documented? And background colors? Do you support italics/bold/flashing/etc?

**Comment by p0nce** [comment]
> Also, why are color names not documented? And background colors? 

You were supposed to run the example, but now there is: 
  - a cheat sheet in README to avoid that. in v1.1.2
  - you can also run the unittest in console-colors


> Do you support italics/bold/flashing/etc?

No, but this is open-source contributions welcome. It should work on Windows, macOS and Linux though else won't be merged.

As for your initial request, I'm not sure how possible it is yet, since you would need state machine for each possible console output.

**Comment by p0nce** [comment]
Now in console-colors v1.2.0 you can output to stderr using your syntax.
Do not use cwrite with something than stdout or stderr though.
It would appreciated if you could test this on Linux or macOS.

**Comment by renatoathaydes** [comment]
Oh thanks :)
I will give it a try.

**Comment by renatoathaydes** [comment]
I ran the example on Mac M1 with the Kitty terminal:

<img width="744" alt="Screenshot 2023-12-27 at 16 06 41" src="https://github.com/p0nce/console-colors/assets/1901277/1ce53309-a1c4-44f0-b69a-4799fef6a191">

I then changed the example to use `stderr.cwriteln` everywhere `cwriteln` was being used, and it worked as expected.
Confirmed that it's sending output to stderr by doing `dub run 2> /dev/null` which does not output anything.

I can do the same on Linux later on if you want.
